.gitignore

```diff
@@ -2,3 +2,5 @@
 /fake-webserver
 /scaler
 /tools/amGithubNotifier/amGithubNotifier
+.idea/
```

Comment:
these are env specific so best to add them to .git/info/exclude

Reply:
I removed this. PTAL.